Assets/General/Scripts/Console/ConsoleController.cs

```diff
@@ -79,8 +93,77 @@ private void ToggleConsole()
         }
     }
 
+    #region FreelookCam
+    /// <summary>
+    /// The player unposseses the character
+    /// and become a free look cam that can
+    /// fly arround
+    /// </summary>
+    private void ToggleFreeLookCam()
+    {
+        if (!_isInDeveloperMode) return;
+
+        if (!_isInFreeLookCamMode)
+        {
+             _spawnedFreeLookCam = EnterFreeLookCam();
+            _toggleFreeLookCamButton.GetComponentInChildren<TMP_Text>().text = "Exit Free Look Cam";
```

Comment:
Can we set this to a variable so we don't have to always use get component in children.

Reply:
That would require more setup in the future if someone else were to work with the dev console. By just getting the button , I have access to everything inside it.

Reply:
Not necessarily. It doesn't need to be a variable accessible to design. We could create a private TMP_Text variable and set it in start